asclepius/README.md

```diff
@@ -82,28 +82,57 @@ Endpoints
 | Returns      | A list of DDI objects; its label, its URI, and the identifiers of the two drugs involved                                         |
 
 #### Example
-
 ##### Request Body
-
 ```json
-{"drugs": ["chebi:421707", "chebi:465284", "dinto:DB00503", "chebi:9342"]}
+{
+  "drugs": [
```

Comment:
Does this need to be drugs or can we just send the list with no object around it? Realise this is an old one but sure